app/components/collections/release_component.rb

```diff
@@ -29,5 +29,17 @@ def release_info
         "depositor selects release date at no more than #{collection.release_duration} from date of deposit"
       end
     end
+
+    sig { returns(String) }
+    def doi_assignment
+      case collection.doi_option
```

Comment:
another way of handling this would have been do `I18n.translate(collection.doi_option, scope: 'collections.release.doi_option')` But this is fine too.